### DISCUSSION TITLE
Setup Heroic Games Launcher with Wineskin

### DISCUSSION THREAD

**Discussion by MattiaFenzi** [Heroic-Games-Launcher/HeroicGamesLauncher#808] [Q&A]
Hi, how can I use Heroic Games Launcher with Wineskin? I tried selecting the .app wrapper that Wineskin creates but the launcher doesn't find any custom Wine
PS: I'm on a Mac

**Comment by flavioislima** [comment]
I have no idea what is Wineskin, but if it's a wrapper for Wine on Mac, Heroic won't be able to use it.
it works with crossover because the guys from codeweavers have added support in heroic.

**Comment by MattiaFenzi** [comment]
So I have to download wine without any wrappers?

**Reply by flavioislima** [reply]
I recommend using crossover.
